From 3259fd7c6f7186d24c0882484891ccd5dd54ae29 Mon Sep 17 00:00:00 2001
From: Paper <paper@tflc.us>
Date: Mon, 23 Dec 2024 20:12:11 -0500
Subject: [PATCH] win_utf8_io: revert commit e8ffe52

this causes a build error when building under mingw, and schism
doesn't use these functions anyway.
---
 src/share/win_utf8_io/win_utf8_io.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/share/win_utf8_io/win_utf8_io.c b/src/share/win_utf8_io/win_utf8_io.c
index 3ae35b31..3112a1e6 100644
--- a/src/share/win_utf8_io/win_utf8_io.c
+++ b/src/share/win_utf8_io/win_utf8_io.c
@@ -355,13 +355,13 @@ int chmod_utf8(const char *filename, int pmode)
 int utime_utf8(const char *filename, struct utimbuf *times)
 {
 	wchar_t *wname;
-	struct __utimbuf64 ut;
+	struct _utimbuf ut;
 	int ret;
 
 	if (!(wname = wchar_from_utf8(filename))) return -1;
 	ut.actime = times->actime;
 	ut.modtime = times->modtime;
-	ret = _wutime64(wname, &ut);
+	ret = _wutime(wname, &ut);
 	free(wname);
 
 	return ret;
-- 
2.39.5

